docs/monitoring.md

```diff
@@ -0,0 +1,221 @@
+# Enabling monitoring {#ovms_docs_monitoring}
+
+## Introduction to Metrics
+
+This document describes enabling and usage of different metrics in OVMS server that can be used for:
+
+- Providing performance statistics for monitoring and benchmarking purposes
+
+- Ability to auto scale model server instances in Kubernetes based on application related metrics
+
+> **NOTE**: currently, Metrics feature is released as a preview feature.
+
+Internal metrics remove dependencies on the 3rd party components and allow tracking performance without extra communication hop.
+
+It also exposes metrics which are not related to the traffic. 
+
+For example number of Idle execution streams, which models were used, model load time etc, which device was used.
+
+There are four types of metrics: 
```

Comment:
Drop this info about metrics types. Mention here that OpenVINO Model Server metrics are compatible with Prometheus standard https://prometheus.io/docs 

Add also info that metrics are exposed on `metrics` endpoint path.

Reply:
Ok